### PR TITLE
Update website documentation

### PR DESCRIPTION
**Pull request by jirevwe**

### CODE REVIEW

**Review by horlah — APPROVED** [review]
Looks good on this side.

File: web/ui/website/content/docs/deployment.md [hunk 1]
```diff
@@ -25,24 +25,100 @@ Then you can follow these steps:
...
+
+6. Move your config file the folder
+```bash
+$ mv convoy.json /etc/convoy/convoy.json
+```
+
+7. Reload the service
+```bash
+$ systemctl resatrt convoy.service
+```
```
Comment (danvixent) [inline web/ui/website/content/docs/deployment.md:124]: I feel like the 4 should be 6

File: web/ui/website/content/docs/overview.md [hunk 2]
```diff
@@ -1,54 +1,98 @@
...
 
 This section collects brief definitions of some of the technical terms used in the documentation for Convoy.
 
+### Groups
+
+Groups are used to create logical contexts or separate environments (dev, staging & production). Different groups can also be created for different teams each with their own login details on the same convoy deployment.
+
 ### Applications
 
 An application represents a user's application trying to receive webhooks. Once you create an application on Convoy. You receive an `app_id` that you should save and supply in subsequent API calls to perform other actions E.g. Send an event. Currently, an application maps to one endpoint. In the future, an application should map to multiple endpoints. When you creating an application, you should supply a [secret](#secrets).
```
Comment (danvixent) [inline web/ui/website/content/docs/overview.md:23]: ahh, we can map to multiple endpoints.

File: web/ui/website/content/docs/overview.md [hunk 3]
```diff
@@ -1,54 +1,98 @@
...
 
 ### Endpoints
 
-An endpoint represents a target URL to receive events. Endpoint can be in either of this states - `active`, `inactive` or `pending`. When an endpoint is in the `inactive` state all events sent will be saved and discarded until the endpoint is brought back up.
+An endpoint represents a target URL to receive events. An endpoint can be in either of these states - `active`, `inactive` or `pending`. When an endpoint is in the `inactive` state all events sent will be saved but not dispatched until the endpoint is re-enabled. These events are known set to the `Discarded` state.
```
Comment (danvixent) [inline web/ui/website/content/docs/overview.md:27]: The pending state wasn't explained

**Pull request merged**